packages/teleport-plugin-css/src/index.ts

```diff
@@ -21,17 +21,23 @@ interface VueStyleChunkConfig {
   chunkName: string
```

Comment:
is it feasible to use this plugin for JSX based frameworks as well? maybe we can have another config options like `templateType`: 'JSX' | 'HTML'. Just an idea, we shouldn't focus on this now

Reply:
Yeah, I feel we could completely use it. We just need to add a 'flavor` flag and use babel types and traves the tree and inject style tags as we do in react based style flavors 👍  That gives support for CSS by default for all types 🎉